src/Symfony/Component/Yaml/Inline.php

```diff
@@ -684,7 +684,7 @@ public static function evaluateBinaryScalar($scalar)
 
     private static function isBinaryString($value)
     {
-        return !preg_match('//u', $value) || preg_match('/[^\x09-\x0d\x20-\xff]/', $value);
+        return !preg_match('//u', $value) || preg_match('/[^\x00-\x1f\x20-\xff]/', $value);
```

Comment:
Ah okay, maybe we should add a comment with some reference to an external source or something like that to help understanding the code.